src/renderer/components/+welcome/welcome.tsx

```diff
@@ -29,46 +29,66 @@ import { WelcomeMenuRegistry } from "../../../extensions/registries";
 
+    // if there is banner with specified width, use it to calculate the width of the container
+    const maxWidth = welcomeBanner.reduce((acc, curr) => {
+      const currWidth = curr.width ?? 0;
+
+      if (acc > currWidth) {
+        return acc;
+      }
+
+      return currWidth;
+    }, defaultWidth);
+
     return (
       <>
         <WelcomeTopbar/>
         <div className="flex justify-center Welcome align-center">
-          <div className="box">
+          <div style={{ width: `${maxWidth}px` }} data-testid="welcome-banner-container">
             {welcomeBanner.length > 0 ? (
               <Carousel
                 stopAutoPlayOnHover={true}
                 indicators={welcomeBanner.length > 1}
                 autoPlay={true}
+                navButtonsAlwaysInvisible={true}
               >
                 {welcomeBanner.map((item, index) =>
                   <item.Banner key={index} />
                 )}
               </Carousel>
             ) : <Icon svg="logo-lens" className="logo" />}
 
-            <h2>Welcome to {productName} 5!</h2>
+            <div className="flex justify-center">
+              <div style={{ width: `${defaultWidth}px` }} data-testid="welcome-text-container">
+                <h2>Welcome to {productName} 5!</h2>
 
-            <p>
+                <p>
               To get you started we have auto-detected your clusters in your kubeconfig file and added them to the catalog, your centralized view for managing all your cloud-native resources.
-              <br/><br/>
+                  <br /><br />
               If you have any questions or feedback, please join our <a href={slackUrl} target="_blank" rel="noreferrer" className="link">Lens Community slack channel</a>.
-            </p>
+                </p>
 
-            <ul className="box">
-              {WelcomeMenuRegistry.getInstance().getItems().map((item, index) => (
-                <li key={index} className="flex grid-12" onClick={() => item.click()}>
-                  <Icon material={item.icon} className="box col-1" /> <a className="box col-10">{typeof item.title === "string" ? item.title : item.title()}</a> <Icon material="navigate_next" className="box col-1" />
-                </li>
-              ))}
-            </ul>
+                <ul className="block" style={{ width: `${defaultWidth}px` }} data-testid="welcome-menu-container">
+                  {WelcomeMenuRegistry.getInstance().getItems().map((item, index) => (
+                    <li key={index} className="flex grid-12" onClick={() => item.click()}>
+                      <Icon material={item.icon} className="box col-1" /> <a className="box col-10">{typeof item.title === "string" ? item.title : item.title()}</a> <Icon material="navigate_next" className="box col-1" />
+                    </li>
+                  ))}
+                </ul>
+              </div>
+            </div>
           </div>
         </div>
       </>
     );
   }
 }
+
+export { defaultWidth };
```

Comment:
why not just use `export const` above?

Reply:
Old habit to export at the bottom...  I will change.

Reply:
Done https://github.com/lensapp/lens/pull/3663/commits/ae30eb3aeb74b8e9146296358859e8704df8883e